ee/clickhouse/views/insights.py

```diff
@@ -68,10 +69,15 @@ def calculate_path(self, request: Request) -> Dict[str, Any]:
         team = self.team
         filter = PathFilter(request=request, data={"insight": INSIGHT_PATHS})
 
+        funnel_filter = None
+        funnel_filter_data = request.GET.get("funnel_filter")
+        if funnel_filter_data:
+            funnel_filter = Filter(data={"insight": INSIGHT_FUNNELS, **json.loads(funnel_filter_data)})
+
         #  backwards compatibility
         if filter.path_type:
             filter = filter.with_data({PATHS_INCLUDE_EVENT_TYPES: [filter.path_type]})
-        resp = ClickhousePaths(filter=filter, team=team).run()
+        resp = ClickhousePaths(filter=filter, team=team, funnel_filter=funnel_filter).run()
```

Comment:
Reminder to self for future PR: Need to add funnel_filter in for Path Persons too.